relay/polkadot/src/ah_migration/phase1.rs

```diff
@@ -79,54 +79,43 @@ pub fn call_allowed_status(call: &<Runtime as frame_system::Config>::RuntimeCall
 	const OFF: bool = false;
 
 	match call {
-		System(..) => (ON, ON),
-		Scheduler(..) => (OFF, OFF),
-		Preimage(..) => (OFF, OFF),
-		Babe(..) => (ON, ON), // TODO: @muharem double check
-		Timestamp(..) => (ON, ON),
-		Indices(..) => (OFF, OFF),
-		Balances(..) => (OFF, ON),
-		// TransactionPayment has no calls
-		// Authorship has no calls
+		System(..) => (ON, ON), // Remarks, root calls and `set_code` if we need for emergency.
+		Scheduler(..) => (OFF, OFF), // Only for governance, hence disabled.
+		Preimage(..) => (OFF, OFF), // Only for governance, hence disabled.
+		Babe(..) => (ON, ON),   // OK? WHY?
+		Timestamp(..) => (ON, ON), // only `set` inherit
+		Indices(..) => (OFF, OFF), // Not needed anymore and migrated to AH.
+		Balances(..) => (OFF, ON), // Disabled during migration to avoid confusing externals.
```

Comment:
How would fees be paid for any extrinsic (that is allowed) during migration?

Reply:
The fee payment calls into the balances pallet through traits - not calls. So that should still work.